workspaces/marketplace/plugins/marketplace/src/plugin.ts

```diff
@@ -122,19 +139,9 @@ export const DynamicMarketplacePluginContent = marketplacePlugin.provide(
 /**
  * @public
  */
-export const InstallationContextProvider = marketplacePlugin.provide(
```

Comment:
I know we discussed backward compatible today and I said I wouldn't be too much worried about it.

But can we export a null component here maybe? It was an old idea from @debsmita1 and it doesn't cost anything ;)

```tsx
/**
 * @public
 * @deprecated
 */
export const InstallationContextProvider = marketplacePlugin.provide(
  createComponentExtension({
    name: 'InstallationContextProvider',
    component: {
      sync: () => null,
    },
  }),
);
```